flume-conf-generator/src/main/java/com/wipro/ats/bdre/fcgen/FlumeConfGeneratorMain.java

```diff
@@ -53,7 +53,7 @@ public static void main(String[] args) throws FileNotFoundException {
                 java.util.Properties flumeProperties = getProperties.getProperties(processInfo.getProcessId().toString(), "flume");
                 Enumeration e = flumeProperties.propertyNames();
 
-                if (flumeProperties.size() != 0) {
+                if (!flumeProperties.isEmpty()) {
                     while (e.hasMoreElements()) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS134)